Hello @maltaesousa,

For the moment, i doesn't work on my linux.
I get the following error/warning messages: 
```
Configuring GeoMapFish project...
OK.
Committing first version...
astuce: Utilisation de 'master' comme nom de la branche initiale. Le nom de la branche
astuce: par défaut peut changer. Pour configurer le nom de la branche initiale
astuce: pour tous les nouveaux dépôts, et supprimer cet avertissement, lancez :
astuce: 
astuce: 	git config --global init.defaultBranch <nom>
astuce: 
astuce: Les noms les plus utilisés à la place de 'master' sont 'main', 'trunk' et
astuce: 'development'. La branche nouvellement créée peut être rénommée avec :
astuce: 
astuce: 	git branch -m <nom>
OK.
Compiling GeoMapFish project...
Traceback (most recent call last):
  File "/tmp/gmf/my-super-gmf-app/./build", line 15, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
OK.
Configuring GeoMapFish Database...
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "POSTGRES_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGPASSWORD" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGDATABASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGUSER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGISSERVER_URL" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGIS_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_INNER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_CONFIG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SIMPLE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_CONFIG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_INNER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGISSERVER_URL" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGIS_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "POSTGRES_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGPASSWORD" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGDATABASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGUSER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SIMPLE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SIMPLE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "POSTGRES_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGPASSWORD" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGDATABASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGUSER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_CONFIG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_INNER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGISSERVER_URL" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGIS_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "SIMPLE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PACKAGE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGISSERVER_URL" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "QGIS_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "POSTGRES_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGPASSWORD" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGDATABASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGUSER" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_CONFIG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "FRONT_INNER_PORT" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_BASE" variable is not set. Defaulting to a blank string. 
WARN[0000] The "DOCKER_TAG" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GIT_HASH" variable is not set. Defaulting to a blank string. 
WARN[0000] The "PGSCHEMA" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_VERSION" variable is not set. Defaulting to a blank string. 
WARN[0000] The "GEOMAPFISH_MAIN_VERSION" variable is not set. Defaulting to a blank string. 
1 error(s) decoding:

* error decoding 'Ports': No port specified: :<empty>
```

I am installing on Manjaro-Linux with all default parameters and with the default database part. No Proxy.

I'm not familiar with 2.7 yet... Any Idea ?

--- a/README.md
+++ b/README.md
@@ -43,7 +43,7 @@ The first step is to install the GeoMapFish requirements.
 You will need to have the following components installed on your system:
 
 - Git
-- Python >= 3.5
+- Python >= 3.7 with pyyaml
 - Docker >= 17.05
 - Docker-Compose
 

--- a/install.sh
+++ b/install.sh
@@ -24,9 +24,21 @@ check()
   if command -v $1 > /dev/null
   then
     version=`$1 --version | head -1 2>&1`
-    echo "${Green}[OK]  $version"
+    echo "${Green}[OK] $version"
   else
     echo "${Red}[NOK] $1 NOT FOUND"
+    echo "Please install the missing requirement."
+    abort 91
+  fi
+}
+
+checkpythonmodule()
+{
+  if python3 -c "import pkgutil; exit(not pkgutil.find_loader('$1'))"
+  then
+    echo "${Green}[OK] $1"
+  else
+    echo "${Red}[NOK] $1 python module NOT FOUND"
     echo "Please install the missing requirement."
     abort 91
   fi
@@ -37,7 +49,7 @@ checkuser()
   user=`whoami`
   if getent group docker | grep -q $user
   then
-    echo "${Green}[OK]  User $user is in group docker"
+    echo "${Green}[OK] User $user is in group docker"
   else
     echo "${Red}[NOK] User $user is not in group docker"
     abort 92
@@ -52,7 +64,7 @@ checkport()
     if [ $used == 0 ]
     then
       gmf_port=$i
-      echo "${Green}[OK]  Port $gmf_port will be used by GeoMapFish"
+      echo "${Green}[OK] Port $gmf_port will be used by GeoMapFish"
       return
     fi
   done
@@ -67,6 +79,7 @@ check 'git'
 check 'docker'
 check 'docker-compose'
 check 'python3'
+checkpythonmodule 'yaml'
 check 'ss'
 check 'sed'
 check 'wget'

--- a/install.sh
+++ b/install.sh
@@ -278,8 +278,13 @@ sed -i "s/PGPASSWORD=<pass>/PGPASSWORD=${dbpass}/g" env.project
 sed -i "s/PGSSLMODE=require/PGSSLMODE=prefer/g" env.project
 sed -i "s/VISIBLE_WEB_HOST=localhost/VISIBLE_WEB_HOST=${gmf_host}/g" env.default
 sed -i "s/8484/${gmf_port}/g" env.default
-start=$(expr $(grep -nE ' {6}service: config' docker-compose.yaml | cut -d : -f 1) + 1)
-sed -i "$start i \    pull_policy: never" docker-compose.yaml
+
+# Do not try to build config on docker-compose v2
+if docker-compose -v | grep -o "v2" > /dev/null
+then
+  start=$(expr $(grep -nE ' {6}service: config' docker-compose.yaml | cut -d : -f 1) + 1)
+  sed -i "$start i \    pull_policy: never" docker-compose.yaml
+fi
 echo "${Green}OK."
 
 # Initialize git and first commit

--- a/install.sh
+++ b/install.sh
@@ -278,6 +278,8 @@ sed -i "s/PGPASSWORD=<pass>/PGPASSWORD=${dbpass}/g" env.project
 sed -i "s/PGSSLMODE=require/PGSSLMODE=prefer/g" env.project
 sed -i "s/VISIBLE_WEB_HOST=localhost/VISIBLE_WEB_HOST=${gmf_host}/g" env.default
 sed -i "s/8484/${gmf_port}/g" env.default
+start=$(expr $(grep -nE ' {6}service: config' docker-compose.yaml | cut -d : -f 1) + 1)
+sed -i "$start i \    pull_policy: never" docker-compose.yaml
 echo "${Green}OK."
 
 # Initialize git and first commit

